api/pom.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>gov.nsa.datawave.microservice</groupId>
+        <artifactId>datawave-microservice-parent</artifactId>
+        <version>4.0.0</version>
+        <relativePath>../../../microservice-parent/pom.xml</relativePath>
+    </parent>
+    <artifactId>file-provider-service-api</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <url>https://github.com/awrigh11/datawave-file-provider-service-awrigh11</url>
+    <scm>
+        <connection>scm:git:https://github.com/awrigh11/datawave-file-provider-service-awrigh11.git</connection>
+        <developerConnection>scm:git:git@github.com:awrigh11/datawave-file-provider-service-awrigh11.git</developerConnection>
+        <tag>HEAD</tag>
+        <url>https://github.com/awrigh11/datawave-file-provider-service-awrigh11</url>
```

Comment:
Is it possible to update the connection information here to be generic?

Reply:
Does the change I made to this file in the latest commit address your concern?

Reply:
Yes, thank you.